docs/introduction.rst

```diff
@@ -318,6 +318,7 @@ an algorithm, which is concerned only with the inputs and outputs.
 Additionally, the algorithm can be flexibly determined by the user; that is, the user is free to choose and configure
 the algorithm as needed for the problem domain.
 
+:ref:`custom text <Class: SlidingWindow>`
```

Comment:
@Purg Realized this snuck in on the PR (seems like a leftover artifact from early debugging tests). Let's try to remove this when we update the saliency algorithms table with the new DRISEStack implementation as part of Elim's PR.